packages/core/admin/admin/src/pages/MarketplacePage/components/SortSelect/index.js

```diff
@@ -1,135 +1,68 @@
-import React, { useRef, useState } from 'react';
+import React from 'react';
+import styled from 'styled-components';
+import { Select, Option } from '@strapi/design-system/Select';
+import { Box } from '@strapi/design-system/Box';
 import { useIntl } from 'react-intl';
 import PropTypes from 'prop-types';
-import { Option } from '@strapi/design-system/Select';
-import { Popover } from '@strapi/design-system/Popover';
-import { Button } from '@strapi/design-system/Button';
-import { FocusTrap } from '@strapi/design-system/FocusTrap';
-import CarretDown from '@strapi/icons/CarretDown';
-import styled from 'styled-components';
 
-const SortOption = styled(Option)`
-  list-style-type: none;
-`;
-
-const SortToggleButton = styled(Button)`
-  svg {
-    width: ${({ theme }) => theme.spaces[2]};
-    height: ${({ theme }) => theme.spaces[1]};
-  }
+const SelectWrapper = styled(Box)`
+  font-weight: ${({ theme }) => theme.fontWeights.semiBold};
 
-  svg > path {
-    fill: ${({ theme }) => theme.colors.neutral500};
+  span {
+    font-size: ${({ theme }) => theme.spaces[3]};
```

Comment:
I think it would be more appropriate to use [fontSizes from the design system theme](https://github.com/strapi/design-system/blob/b9a7e0dac6d2bd0f64852c97b143b16e11752219/packages/strapi-design-system/src/themes/common-theme.js#L12)